tests/failing-symbolic.haskell

```diff
@@ -1,4 +1,3 @@
-tests/specs/benchmarks/bytes00-spec.k
```

Comment:
Do these tests pass in <20m on your machine? If not, it's unlikely they'll make it past CI. We should definitely add the term which is simplified by this lemma to the `tests/specs/benchmarks/functional-spec.k` file, because we won't be able to enable these tests yet (and running that file first to debug updates to KEVM is much easier than running the entire proofs).

Reply:
No, they take closer to 40m. I will remove the tests from `tests/failing-symbolic.haskell` and add that spec to `tests/specs/benchmarks/functional-spec.k` next.